server.js

```diff
@@ -1,16 +1,17 @@
-require("dotenv").config();
+require('dotenv').config();
 
-const express = require("express");
-const path = require("path");
+const express = require('express');
+const path = require('path');
 
-const { PORT } = process.env;
+const { PORT = 3000 } = process.env;
+const HOST = '0.0.0.0';
```

Comment:
why do you have this host here, andif you need it this should obviously be an env variable...